x-pack/legacy/plugins/maps/public/layers/sources/client_file_source/geojson_file_source.js

```diff
@@ -156,4 +138,7 @@ export class GeojsonFileSource extends AbstractVectorSource {
     return GeojsonFileSource.isIndexingSource;
   }
 
+  isPushedData() {
```

Comment:
Don't think sources need to be aware of pushed data. The concept can exist at the layer level